src/ParallelAlgorithms/Amr/Policies/Policies.hpp

```diff
@@ -0,0 +1,47 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "Options/String.hpp"
+#include "ParallelAlgorithms/Amr/Policies/Isotropy.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace amr {
+/// \brief A set of runtime policies controlling adaptive mesh refinement
+class Policies {
+ public:
+  /// The isotropy of adaptive mesh refinement (whether or not each dimension
+  /// can be refined independently)
+  struct Isotropy {
+    using type = amr::Isotropy;
+    static constexpr Options::String help = {"Isotropy of AMR."};
```

Comment:
Move dox string to options help string (the options help is more important that the dox I think. The options help actually appears in the dox.)